package.json

```diff
@@ -1,18 +1,20 @@
 {
-  "name": "Titter-WRIO-App",
+  "name": "titter-wrio-app",
```

Comment:
"New packages must not have uppercase letters in the name."(https://docs.npmjs.com/files/package.json#name)